caps/views.py

```diff
@@ -256,6 +237,173 @@ def get_context_data(self, **kwargs):
         # need to convert to a dict as items doesn't work on defaultdicts
         # in django templates
         context["grouped_documents"] = dict(grouped_documents)
+        return context
+
+    def get_council_card_context(self, context: dict[str, Any]) -> dict[str, Any]:
+        """
+        Return the council cards that are valid for this council
+        """
+
+        class MenuItem(NamedTuple):
```

Comment:
I do wonder if this and the menu structure could be pulled out into another file, I dunno `council_navigation.py` or something and then included. Mostly as this is a huge file already and removing config out of it might be good. Not urgent/important though.